Sources/InputBarAccessoryView.swift

```diff
@@ -584,7 +584,10 @@ open class InputBarAccessoryView: UIView {
                 inputTextView.invalidateIntrinsicContentSize()
             }
         }
-        
+        if let middleContentView, middleContentView != inputTextView {
+            inputTextViewHeight = middleContentView.intrinsicContentSize.height
+        }
```

Comment:
Yea I think this is right. Replacing the middle content view is def very experimental, and this was a missed edge case.

I think we should optimize this tho, so that we skip calculating `inputTextViewHeight` if the middleContentView != inputTextView